dist/scripts/datalayer/common.js

```diff
@@ -1,100 +1,162 @@
+        window.dataLayer.push(e));
     },
     event: {
         click: {
             login: '.woocommerce-form-login button[name="login"]',
             register: '.woocommerce-form-register button[name="register"]',
             registerOnCheckout: ".woocommerce-checkout #place_order"
         }
+    },
+    cart: {
+        elements: {
+            product: ".shop-analytics-product-info",
+            shippingMethods: 'select.shipping_method, input[name^="shipping_method"]',
+            shippingMethodSelected: 'select.shipping_method:selected, input[name^="shipping_method"]:checked',
+            paymentMethods: 'select.payment_method, input[name^="payment_method"]',
+            paymentMethodSelected: 'select.payment_method:selected, input[name^="payment_method"]:checked'
+        }
+    },
+    checkout: {
+        dataInit: {
+            event: "EECcheckout",
+            ecommerce: {
+                checkout: {
+                    actionField: {
+                        step: 0
+                    },
+                    products: []
+                }
+            }
+        },
+        elements: {
+            shippingMethods: 'select.shipping_method, input[name^="shipping_method"]',
+            shippingMethodSelected: 'select.shipping_method:selected, input[name^="shipping_method"]:checked',
+            paymentMethods: 'input[name="payment_method"]',
+            paymentMethodSelected: 'select.payment_method:selected, input[name="payment_method"]:checked',
+            billingAddressFields: ".woocommerce-billing-fields input.input-text, .woocommerce-billing-fields select",
+            billingAddressFieldsRequired: ".woocommerce-billing-fields .validate-required input.input-text, .woocommerce-billing-fields .validate-required select",
+            shippingAddressFields: ".woocommerce-shipping-fields input.input-text, .woocommerce-shipping-fields select",
+            shippingAddressFieldsRequired: ".woocommerce-shipping-fields .validate-required input.input-text, .woocommerce-shipping-fields .validate-required select",
+            shippingAddressToggle: "#ship-to-different-address-checkbox"
+        },
+        messages: {
+            shipToSameAddress: "Ship to same address",
+            shipToDifferentAddress: "Ship to different address"
+        }
     }
-}, function(t) {
-    function e(e, o) {
-        var a = t(o && o.responseText ? o.responseText : document).find(".shop-analytics-product-details");
-        if (a.length && !a.parents(".shop-analytics-order-details").length) {
-            var n = 1, r = "";
-            a.each(function(e) {
-                var o = t(this), a = i.getProductsListType(o);
-                r !== a && (r = a, n = 1), o.data("position", n++), o.data("list", a);
+}, function(e) {
+    function t(t, o) {
+        var i = e(o && o.responseText ? o.responseText : document).find(".shop-analytics-product-details");
```

Comment:
@juanlopez4691 Why are you trying to use the text version of an Ajax response as a DOM document?

This causes a JavaScript error during WooCommerce checkout, as this code attempts to parse the JSON response from the backend into a jQuery context:
<img width="1440" alt="shop-analytics woocommerce-checkout-js-error" src="https://user-images.githubusercontent.com/41992/47080782-30cbc780-d209-11e8-8114-8941e220cc05.png">

Reply:
@sun created a new PR to do a quick fix on this: https://github.com/netzstrategen/wordpress-shop-analytics/pull/15